scene/gui/dialogs.cpp

```diff
@@ -395,12 +395,15 @@ AcceptDialog::AcceptDialog() {
 	bg_panel = memnew(Panel);
 	add_child(bg_panel, false, INTERNAL_MODE_FRONT);
 
+	scroll_bar = memnew(ScrollContainer);
```

Comment:
Calling it scroll bar but instantiating a scroll container probably isn't a good idea, considering ScrollBar is a call that exists jn the API. I recommend updating this member name.